.maintain/sentry-node/docker-compose.yml

```diff
@@ -131,11 +131,6 @@ services:
       - "sub-authority-discovery=trace"
       - "--prometheus-external"
 
-  ui:
-    image: polkadot-js/apps
```

Comment:
Also happy to replace this with https://hub.docker.com/r/chevdor/polkadot-ui in case that is the image we recommend to users.

Reply:
Or https://hub.docker.com/r/jacogr/polkadot-js-apps ?